test/Microsoft.AspNetCore.ChunkingCookieManager.Sources.Test/CookieChunkingTests.cs

```diff
@@ -5,8 +5,13 @@
 using Microsoft.AspNetCore.Http;
 using Xunit;
 
-namespace Microsoft.AspNetCore.Authentication.Cookies.Infrastructure
+#if SECURITY
```

Comment:
You don't need an if here, this file doesn't move, you will only ever be testing one version of the src file.